runtime/amplitude/src/lib.rs

```diff
@@ -148,62 +151,42 @@ pub type SignedPayload = generic::SignedPayload<RuntimeCall, SignedExtra>;
 /// Extrinsic type that has already been checked.
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
-use crate::sp_api_hidden_includes_construct_runtime::hidden_include::dispatch::GetStorageVersion;
-use frame_support::pallet_prelude::StorageVersion;
+use runtime_common::asset_registry::StringLimit;
 
 parameter_types! {
 	pub const InactiveAccounts: Vec<AccountId> = Vec::new();
 }
 
-// Temporary struct that defines the executions to be done upon upgrade,
-// Should be removed or at least checked on each upgrade to see if it is relevant,
-// given that these are "one-time" executions for particular upgrades
+// TODO remove contract migrations after update
```

Comment:
After update of? Version 1.1.0?

Reply:
Yes! It does not necessarily mean that the storage migrations are to be coupled with an upgrade, but it will generally be the case. 

As soon as any runtime upgrade is executed on chain with this code declared, the migrations will happen, and this code becomes irrelevant to keep.

Reply:
In theory these are tested for idempotence, such that if we forget and upgrade many times with this code still there it will not affect the storage. But I think it is better to remove.